requirements.txt

```diff
@@ -2,6 +2,7 @@ adal==0.4.0
 applicationinsights==0.10.0
 argcomplete==1.3.0
 azure==2.0.0rc5
+azure-mgmt-trafficmanager==0.30.0rc5
```

Comment:
nope, they have it separated out until it becomes stable (feature is in preview right now)

---
In reply to: [75039967](https://github.com/Azure/azure-cli/pull/708#discussion_r75039967) [](ancestors = 75039967)